portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,42 +13,92 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import com.google.sps.data.Task;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import com.google.gson.Gson;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that handles comment data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
-  private List<String> facts;
+  private ArrayList<String> comments;
+  private static final Gson gson = new Gson();
```

Comment:
nit: we should change this to "GSON" (all caps variable name) since it's constant:
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names